app/src/components/card/searchCard.tsx

```diff
@@ -0,0 +1,138 @@
+"use client";
+import React from "react";
+import {
+  Card,
+  CardHeading,
+  CardContent,
+  StatusBadge,
+  TagSet,
+  Flex,
+  Text,
+  Box,
+} from "@nypl/design-system-react-components";
+import SearchCardType, {
+  SearchResultRecordType,
+} from "@/src/types/SearchCardType";
+
+export interface SearchCardProps {
+  result: SearchCardType;
+  keywords: string;
+}
+
+const onSiteMaterialBadge = (recordType: SearchResultRecordType) => {
+  return (
+    <StatusBadge sx={{ margin: "0p" }} type="informative">
+      {recordType === "Item"
+        ? "Available onsite only"
+        : "Contains on-site materials"}
+    </StatusBadge>
+  );
+};
+
+const contentTypeTag = (result: SearchCardType) => {
+  const displayLabel =
+    result.recordType === "Item"
+      ? result.contentType === "Image" && result.containsMultipleCaptures
+        ? "Multiple images"
+        : result.contentType
+      : result.recordType;
+
+  return (
+    <TagSet
+      onClick={() => {}}
+      tagSetData={[
+        { id: `type-${result.uuid}`, label: displayLabel ? displayLabel : "" },
+      ]}
+      type="filter"
+      sx={{ margin: 0 }}
+    />
+  );
+};
+
+const highlightedText = ({ highlight, keyword }) => {
+  const words = highlight.text.split(" ");
+  const keywords = keyword.split(" ");
+  return (
+    <Box
+      sx={{
+        [`@media screen and (max-width: 600px)`]: {
+          display: "none",
+        },
+      }}
+    >
+      <Text
+        sx={{
+          fontWeight: "400",
+          margin: 0,
+          display: "inline",
+        }}
+      >
+        {highlight.field}:{" "}
+      </Text>
+      {words.map((word: string, index: number) => {
+        const isKeyword = keywords.some(
+          (keyword: string) => keyword.toLowerCase() === word.toLowerCase()
+        );
+
+        return (
+          <span key={index}>
+            {isKeyword ? (
+              <Text
```

Comment:
This is rendering a `p` inside of a `span`. Not technically wrong but better to not have block elements inside inlined ones. You can add `as="span"` and then remove the "display" rule below.

Reply:
This is still a concern.